radiography/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ android {
   }
 
   defaultConfig {
-    minSdkVersion(17)
+    minSdkVersion(21)
```

Comment:
This gets exported in our AAR, right? Bumping it reduces the uses for radiography. Is there a good reason here?